Khan/wonder-blocks
# Add controls for LabeledTextField to storybook

## Summary:
Add controls for LabeledTextField in storybook and add
descriptions for each story.

I also made a few small changes to TextField stories
as I was using them as a reference and figured I'd
udpate both at the same time when I spotted
areas of improvement.

I added an "Error Light" story to both `LabeledTextField` and
`TextField` in order to view the light error style.

View the following Storybook publish to see changes:

https://5e1bf4b385e3fb0020b7073c-gndecaotnq.chromatic.com/?path=/docs/form-labeledtextfield--default


Issue: https://khanacademy.atlassian.net/browse/WB-1295

## Test plan:
- Go to http://localhost:6061/?path=/docs/form-labeledtextfield--default
- Check that the description and usage info are there.
- Check that the default story is at the top.
- Read through the props and confirm that they all look right.
- Read through the story descriptions and interact with them
  all to confirm that they all look and behave as expected.
